### DISCUSSION TITLE
Is it possible to add fields via twig?

### DISCUSSION THREAD

**Discussion by mprofitlich** [verbb/formie#1633] [General] [answered]
Building a low-level shop I want the frontend to show products. For each product an amount can be selected. My idea is to dynamically add number fields via twig, based on a Craft section. Is that possible?

Thanks for your thoughts!

<img width="636" alt="grafik" src="https://github.com/verbb/formie/assets/6146601/b1c387cb-90cf-4a7f-9f98-4fe785e5d4d1">

**Comment by engram-design** [comment, marked as answer]
I'd say Formie isn't a great fit for this sort of thing, sorry! This looks like something you'd need to build yourself, and you certainly won't be able to dynamically add fields to a form via Twig.

**Reply by mprofitlich** [reply]
Thank you for your reply. It's good to know that this is impossible – I can now bury this  approach and focus on the other one I have. Thank you!
